arangod/RocksDBEngine/RocksDBMetaCollection.h

```diff
@@ -192,7 +191,7 @@ class RocksDBMetaCollection : public PhysicalCollection {
  protected:
   RocksDBMetadata _meta;  /// collection metadata
   /// @brief collection lock used for write access
-  mutable basics::ReadWriteLock _exclusiveLock;
+  mutable std::shared_timed_mutex _exclusiveLock;
```

Comment:
If we are using `shared_timed_mutex` now, we must also include `shared_mutex` in the header.

Reply:
sure.